ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -95,3 +95,90 @@ keys:
                         type: int
                         convert_types:
                           - str
+                  police:
+                    type: dict
+                    keys:
+                      cir:
+                        type: dict
+                        description: Set committed information rate.
+                        keys:
+                          cir_rate:
+                            required: true
+                            type: int
+                            convert_types:
+                              - str
+                            description: |
+                              Specify lower rate.
+                              Lower rate range in kbps - <8-200000000>.
```

Comment:
I think this description was copied from lower_rate. Please update.

Reply:
I found it same in EOS

Reply:
Well it is misleading for this field.

Reply:
removed the CIR